src/nginxconfig/templates/setup.vue

```diff
@@ -150,11 +150,10 @@ THE SOFTWARE.
                 return this.tarContents().base64(path);
             },
             setupCopy(elm) {
-                const originalText = elm.textContent;
 
                 const resetText = () => {
                     setTimeout(() => {
-                        elm.textContent = originalText;
+                        elm.textContent = this.$t('templates.setup.copyBase64');
```

Comment:
👋 `setupCopy` gets used in a couple of different places, so we can't rely on hard-coding a specific string here.

What'd I recommend doing is in the `success` and `error` handlers below, making a copy of the original text there, then editing changing the text, and passing the original into this `resetText` function.

Reply:
Thank you for your great idea, I hope that with this updating I can resolve this bug